zerver/lib/narrow.py

```diff
@@ -157,6 +157,12 @@ def narrow_filter(event: Mapping[str, Any]) -> bool:
                 if operand.lower() != message["sender_email"].lower():
                     return False
             elif operator == "is" and operand == "private":
+                # "private" is a deprecated operand with the "is" operator.
```

Comment:
Somehow I feel like this comment would be more readable if we just said "# is:private is a legacy alias for is:dm".